### ISSUE TITLE
Compiler error

### ISSUE THREAD

**Issue by soerent** [usr-lab/PepperSocial#9]
Hi there

When compiling your project (MacOS) I get the following error:
Library/PackageCache/com.unity.timeline@1.2.6/Editor/Window/TimelineWindow_Breadcrumbs.cs(85,46): error CS0433: The type 'ValueTuple<T1, T2>' exists in both 'System.ValueTuple, Version=4.0.1.1, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'

Any ideas why?

**Comment by mightypirate1** [comment]
Not really. My very limited experience with MacOS just tells me things are
usually more difficult.

Does UnityML in general work on your system?

Note that the repo in question is forked off of an old version of UnityML,
so that is the version you would have to get running.
I recommend trying it on Ubuntu, as that is what it was written for / built
on!

It looks like the file that is throwing your error is a file that was not
customized by us on our fork, so I would guess you have a more general
Unity problem.
